subprojects/docs/src/samples/java/multiproject/settings.gradle

```diff
@@ -1,3 +1,4 @@
+rootProject.name = 'multiproject'
```

Comment:
This may be due to the way sample-check copies sample projects to a temporary location. If I omit this, then the `listProjects` sample fails with:

> Expected: Root project 'multiproject'
Actual: Root project 'java_multiproject_listProjects.sample'

Reply:
this is a problem in exemplar, then. It should preserve the directory name when creating the copy.